conftest.py

```diff
@@ -326,6 +351,13 @@ def pytest_cmdline_main(config):
     if upgrade_option == "cnv" and config.getoption("cnv_source") and not config.getoption("cnv_version"):
         raise ValueError("Running with --cnv-source: Missing --cnv-version")
 
+    if conformance_storage_class := config.getoption("conformance_storage_class"):
+        if config.getoption("storage_class_matrix"):
+            raise ValueError("`--storage-class-matrix` and `--conformance-storage-class` are mutually exclusive.")
+
+        if config.getoption("default_storage_class"):
+            LOGGER.warning(f"Using default storage class {conformance_storage_class}")
+
```

Comment:
Should we set `py_config["default_storage_class"]` here? 
Maybe we need to change something in `config_default_storage_class` function?

Reply:
added under `pytest_configure`:
"""
        py_config["default_storage_class"] = conformance_storage_class
"""
will test it